c-api/dotnet/src/Picky/Native/NativeInheritanceAttribute.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Diagnostics;
+
+namespace Devolutions.Picky.Native
+{
+    /// <summary>Defines the base type of a struct as it was in the native signature.</summary>
+    [AttributeUsage(AttributeTargets.Struct, AllowMultiple = false, Inherited = true)]
+    [Conditional("DEBUG")]
+    internal sealed partial class NativeInheritanceAttribute : Attribute
+    {
+        private readonly string _name;
```

Comment:
nit: Actually I prefer the `_` prefix for fields instead of using `this.`. Again, I say do what you like best but StyleCop will probably complain about this if you use it in the future.

Reply:
I can definitely change that (I remember doing something similar in C++ with a `m_` prefix)